gf-webapp/src/logic/bookingPage/form/Form.tsx

```diff
@@ -71,8 +64,8 @@ export const Form = (props: RoomProps) => {
             className={styles.input}
             id="email"
```

Comment:
why do we need id here?

Reply:
Replaced input inside label and deleted id